agent/custom/general_navigation_action.py

```
@@ -237,8 +339,28 @@ def __init__(self, model_path):
         from ultralytics import YOLO
         self.model = YOLO(model_path)
 
+    @staticmethod
+    def _norm_img(img):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `_norm_img`

(ANN205)